src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReferenceTransactionNumberTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ReferenceTransactionNumberTest {
+    @Test
+    public void testValidReferenceTransactionType() throws EdifactValidationException {
+        ReferenceTransactionNumber referenceTransactionNumber =
```

Comment:
Because we are initialising it in every test inside this class, do you think that it might be a good idea to do it in @BeforeEach maybe? Tell me what you think. Thanks!

Reply:
Agreed and done in 41a7a3e